test/e2e_node/node_shutdown_linux_test.go

```diff
@@ -576,13 +581,16 @@ func getGracePeriodOverrideTestPod(name string, node string, gracePeriod int64,
 					Image:   busyboxImage,
 					Command: []string{"sh", "-c"},
 					Args: []string{`
-_term() {
-	echo "Caught SIGTERM signal!"
-	while true; do sleep 5; done
-}
-trap _term SIGTERM
-while true; do sleep 5; done
-`},
+					sleep 9999999 &
```

Comment:
there's `sleep infinity` if you want to use that - https://github.com/search?type=code&auto_enroll=true&q=%22sleep+infinity%22+language%3AShell&l=Shell